packages/serverpod/lib/src/cache/local_cache.dart

```diff
@@ -81,24 +87,47 @@ class LocalCache extends Cache {
   @override
   Future<T?> get<T extends SerializableModel>(
     String key, [
+    /// Handler to generate a new value in case there is no active value in the cache
+    ///
+    /// In case a value computation from a previous [get] call is already running, the caller will receive the value from
```

Comment:
Seems like an important edge-case to mention, though I would think that all calls to `get` for a given type `T` will always use the same `cacheMissHandler`.

(Would be interesting to have an "overlay"/"view" on the cache to have everything cast to `T` and thus also avoid the duplication of lifetimes in `put` and `cacheMissHandler` as I would assume users would always want those to be aligned – but alas just my thoughts of the moment and outside the scope of the bug.)